server/ansible/roles/pbench-server-crontab-disable/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: "disable crontab"
+  command:
+    cmd: crontab -r
```

Comment:
Shouldn't we be specifying a user name (either with a `-u` switch on the command or in a `user:` entry in the YAML)?

Reply:
Probably: the role assumes that it is run under the "correct" identity - it is used by the disable playbook which specifies  `remote_user: pbench` explicitly, but that's a hidden coupling.

Reply:
My principal concern is that `crontab -r` is kind of a big hammer, and I don't want it to land on anyone's thumb.

Reply:
We could run the playbook as root and use -u with the appropriate user, but we were trying to get away from running them as root if we don't have to. So maybe check the id of the user running it and only do it if it is the "right" user?

Reply:
I have never been comfortable with automated installs of crontabs via Ansible.

It is probably worth issuing a `crontab -l` first to save the existing crontab locally before performing a `crontab -r` here.

Reply:
Yes, I was afraid that `crontab -u` was a privileged command, but I couldn't easily prove that.  However, adding the `-u` alone isn't much of a protection.  Backing up the previous contents is a much better solution.

Reply:
> maybe check the id of the user running it and only do it if it is the "right" user?

I'm a big fan of assertion checks, and of detecting inconsistencies and preventing problems, particularly ones which are hard to diagnose after the fact.  So, I would be in favor of adding a check here, or at least an alert if we're not in the right context.  On the other hand, I'm not closely familiar with the operating environment here -- if it's structurally "impossible" to be running under the wrong username, then we can settle for just the strategy of creating a backup (in addition to enabling recovery, that will leave a trace of who actually ran the command and when).

Reply:
Bump (so this thread is attached to my review).